libs/knowledge-store/ragstack_knowledge_store/graph_store.py

```diff
@@ -90,12 +121,18 @@ def _deserialize_links(json_blob: Optional[str]) -> Set[Link]:
 
 
 def _row_to_node(row: Any) -> Node:
-    metadata = _deserialize_metadata(row.metadata_blob)
+    metadata_s = row.metadata_s
```

Comment:
I don't think this should read metadata_s to create the node. Also, we may want to support multiple versions of this:

`map<str, str>` supports basic equality on strings, integers (their string representation, etc.). But I could see `map<str, int>` to support comparisons on the number, or `map<str, set<str>>` to support `contains` on the fields, etc.

Reply:
fixed